pkg/resourcemanager/azuresql/azuresqluser/azuresqluser.go

```diff
@@ -130,8 +130,8 @@ func (s *AzureSqlUserManager) CreateUser(ctx context.Context, secret map[string]
 	return newUser, nil
 }
 
-// UpdateUser - Updates user password
-func (s *AzureSqlUserManager) UpdateUser(ctx context.Context, secret map[string][]byte, db *sql.DB) error {
+// RotateUserPassword - Generate and save a new password for the user
+func (s *AzureSqlUserManager) RotateUserPassword(ctx context.Context, secret map[string][]byte, db *sql.DB) error {
```

Comment:
This function should be defined in the manager as well. I know that UpdateUser previously wasn't, but can you add that chance?